config/index.js

```diff
@@ -360,6 +360,38 @@ var conf = convict({
       doc: 'which key prefix to store sessions in redis under'
     }
   },
+  devicesCache: {
+    expiresIn: {
+      default: '1 hour',
+      format: 'duration',
+      env: 'CACHE_EXPIRES_IN',
```

Comment:
can we prefix all these `env` variables in this section with `DEVICES_CACHE` 

In some cases you can make them shorter, -> `DEVICES_CACHE_ENABLED` instead of `USE_REDIS_FOR_DEVCES_CACHE`